### DISCUSSION TITLE
Import is missing in sample code on Automap documentation

### DISCUSSION THREAD

**Discussion by Flyukiy** [sqlalchemy/sqlalchemy#11267] [Usage Questions] [answered]
Hi, it's a very minor issue I just encountered while testing the automap feature.
Link: [Custom Relationship Arguments](https://docs.sqlalchemy.org/en/20/orm/extensions/automap.html#custom-relationship-arguments)
The `interfaces` import is missing, import line should be `from sqlalchemy.ext.automap import generate_relationship, interfaces`
Thanks!

**Comment by CaselIT** [comment, marked as answer]
Hi,

Thanks for reporting. Feel free to just send PR for similar things
